validation/src/test/scala/hmda/validation/rules/lar/quality/Q035Spec.scala

```diff
@@ -0,0 +1,39 @@
+package hmda.validation.rules.lar.quality
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.{ BadValueUtils, LarEditCheckSpec }
+import org.scalacheck.Gen
+
+class Q035Spec extends LarEditCheckSpec with BadValueUtils {
```

Comment:
Why does this spec extend `BadValueUtils` if it doesn't use it? Better not to import methods you aren't going to use.